docs/google/best_practices.md

```diff
@@ -263,12 +263,62 @@ commuting operators.
+two-qubit gates are measured as a side effect of running the device's
+calibration procedure.  These metrics can be used as a baseline to evaluate
+circuit performance or identify outliers to avoid.  This data can be inspected
+programmatically by retrieving metrics from the [API](calibration.md) or
+visually by applying a `cirq.Heatmap` to that data or by using the built-in
+heatmaps in the Cloud console page for the processor.  Note that, since this
+data is only taken during calibration (e.g. at most daily), drifts and other
+concerns may affect the values significantly, so these metrics should only be
+used as a first approximation.  There is no substitute for actually running
+on the device.
+*   Loschmidt echo:  Running a small circuit on a string of qubits and then
+applying the circuit's inverse can be used as a quick but effective way to
+judge qubit's effectiveness.  See
+[this tutorial](../tutorials/google/echoes.ipynb) for instructions.
+*   XEB:  Cross-entropy benchmarking is another way to gauge qubit's performance
+on a set of random circuits.  See tutorials on
+[parallel XEB](../qcvv/parallel_xeb.ipynb)
+or [isolated XEB](../qcvv/parallel_xeb.ipynb) for instructions.
+
+
 ### Refitting gates
 
 Virtual Z gates (or even single qubit gates) can be added to adjust for errors
 in two qubit gates.  Two qubit gates can have errors due to drift, coherent
-error, or other sources.  Refitting these gates and adjusting the circuit for
-the observed unitary of the two qubit gate compared to the ideal unitary can
-substantially improve results.  However, this approach can use a substantial
-amount of resources, and the methods to do this are currently under active
-research and change.  Talk to your Google sponsor for details.
+error, unintended cross-talk, or other sources.  Refitting these gates and
+adjusting the circuit for the observed unitary of the two qubit gate
+compared to the ideal unitary can substantially improve results.
+However, this approach can use a substantial amount of resources.
+
+This technique involves two distinct steps.  The first is *characterization*,
+which is to identify the true behavior of the two-qubit gate.  This typically
+involves running many varied circuits involving the two qubit gate in a method
+(either periodic or random) to identify the parameters of the gate's behavior.
+
+Gates used in Google's architecture fall into a general category of FSim gates,
+standing for *Fermion simulation*.  The generalized version of this gate can
+be parameterized into 5 angles, or degrees of freedom.  Characterization will
+attempt to identify the values of these five angles.
+
+The second step is calibrating (or refitting) the gate.  For the five angles
+that comprise the generalized FSim gate, three can be corrected for by adding
+Z rotations before or after the gate.  Since these gates are propagated forward
+automatically, they add no duration or error to the circuit and can essentially
```

Comment:
I think we could be a bit more precise what we mean by "propagated forward automatically".  Also maybe we can mention that these are partial Z phases...if I understand correctly - or are these full Z gates?

Reply:
Linked to device page, and also updated that page with more information, since it has been a source of quite some confusion.

Reply:
Did you push this change? I don't see the update neither here nor in the other page.